docs/integrations/webhooks/inngest.mdx

```diff
@@ -0,0 +1,215 @@
+---
+title: Handling webhooks with Inngest
+description: Learn how to integrate Clerk with Inngest.
+---
+
+# Handling webhooks with Inngest
+
+Webhooks allow you to [synchronize data](/docs/users/sync-data) from Clerk to your application backend. You can either handle them directly in your backend with an endpoint or use a tool like [Inngest](https://inngest.com/) which receives the webhook events for you and reliably executes functions in your codebase. When handling webhooks, Inngest receives the webhook events for you and uses a built-in queue to reliably execute longer running functions with additional functionality including:
+
+* [Limiting concurrency](https://www.inngest.com/docs/guides/concurrency) to handle spikes in events without overwhelming your API or database.
+* Triggering multiple functions from a single event ([fan-out jobs](https://www.inngest.com/docs/guides/fan-out-jobs)).
+* [Delaying code](https://www.inngest.com/docs/functions/multi-step) to run after a period of time.
+* [Debouncing events](https://www.inngest.com/docs/reference/functions/debounce) to reduce duplicate processing.
+
+In this guide, you'll learn how to set up Inngest to receive Clerk webhook events and how to define Inngest functions in your application using Clerk events including synchronizing data and sending welcome emails.
+
+To follow this guide, you need an Inngest account and have Inngest [set up](https://www.inngest.com/docs/quick-start) in your codebase.
+
+## Setting up the Inngest webhook
+
+To create an Inngest webhook endpoint and add it to your Clerk account, open the [Webhooks](https://dashboard.clerk.com/last-active?path=webhooks) page at the Clerk dashboard. Next, select the **Add Endpoint** button.
+
+<Images
+  width={2578}
+  height={1652}
+  src="/docs/images/integrations/inngest/webhook-page.webp"
+  alt="The Webhooks page in the Clerk Dashboard. A red arrow points to the button for Add Endpoint."
+/>
+
+On the next page, select the **Transformation template** tab and the **Inngest** template, then click on the **Connect to Inngest** button.
+
+<Images
+  width={1906}
+  height={1376}
+  src="/docs/images/integrations/inngest/webhook-transformation-template.webp"
+  alt="The Webhooks page in the Clerk Dashboard showing the Inngest transformation template. Red arrows point to the Transformation Template tab, the Inngest template, and the Connect to Inngest button."
+/>
+
+A popup window will appear to complete the setup. Select **Approve** to create the webhook.
+
+<Images
+  width={1880}
+  height={1448}
+  src="/docs/images/integrations/inngest/webhook-endpoint-connected.webp"
+  alt="The Webhooks page in the Clerk Dashboard showing a connected Inngest account. A red arrow points to the Connected button."
+/>
```

Comment:
This image appears to be in the wrong order. Looks like it should be flipped with the one below it. 
```suggestion
<Images
  width={1194}
  height={1520}
  src="/docs/images/integrations/inngest/inngest-permissions-dialog.webp"
  alt="The Inngest permissions popup window showing the Approve button."
/>
```